doc/api/stream.md

```diff
@@ -1492,6 +1492,8 @@ changes:
     [`stream._destroy()`][writable-_destroy] method.
   * `final` {Function} Implementation for the
     [`stream._final()`][stream-_final] method.
+  * `autoDestroy` {boolean} Whether this stream should automatically call
+    .destroy() on itself after ending.
```

Comment:
`.destroy()` -> `` `.destroy()` ``?

Reply:
Can you provide the default value explicitly here?